app/src/test/groovy/gov/hhs/cdc/trustedintermediary/external/reportstream/ReportStreamLabOrderSenderTest.groovy

```diff
@@ -18,11 +101,10 @@ class ReportStreamLabOrderSenderTest extends Specification{
                 }
 
         when:
-
+        mockFhir.encodeResourceToJson(_ as String) >> "Mock order"
```

Comment:
This line seems to be setting up the test, so it should be in the `given` block.

Reply:
done.